axe-testing/eslint.config.js

```diff
@@ -0,0 +1,7 @@
+import globals from 'globals';
+import pluginJs from '@eslint/js';
+
+export default [
+  { languageOptions: { globals: globals.browser } },
+  pluginJs.configs.recommended,
+];
```

Comment:
Sounds good!  I tried to moved eslint back to the <9 version and copied the .eslintrc from the api (and installed the same eslint dev dependencies, but <9 is deprecated so followed the instructions to migrate (so now it has the eslint.config.mjs again that extends from the root (but left the old .eslintrc, as well. )  It doesn't work as there are now version conflicts with the eslint at the root... will see if the cloudbuild linting still works.  Should I migrate everything over, or leave this for now, and have migration as a separate activity?

Reply:
Ah, hm, leave for now in that case. Upgrading everything to eslint v9 has been held up for a while, see the comments in #229.

Although now that I'm looking at that PR, I think the last plugin I was waiting on/expecting to get v9 compatibility has updated since I last checked. Eitherway, it should be its own PR.